legend-depot-server/src/main/java/org/finos/legend/depot/server/resources/ProjectsVersionsResource.java

```diff
@@ -44,20 +48,23 @@ public class ProjectsVersionsResource extends BaseResource
 {
 
     private final ProjectsService projectVersionApi;
+    private final InfoResource infoResource;
```

Comment:
a resource cant depend on another resource, there must be  a service with that info.use that or expose that info